maas-region/src/helper.py

```diff
@@ -194,6 +194,22 @@ def setup_region(maas_url: str, dsn: str, mode: str) -> None:
         ]
         subprocess.check_call(cmd)
 
+    @staticmethod
+    def is_tls_enabled() -> bool | None:
+        """Check whether MAAS currently has TLS enabled.
+
+        Returns:
+            bool | None: True if MAAS has TLS enabled, False if not, None if MAAS is not initialized
+        """
+        maas = SnapCache()[MAAS_SNAP_NAME]
+        nginx_cfg_filename = f"/var/snap/maas/{maas.revision}/http/regiond.nginx.conf"
```

Comment:
Is the usage of current an issue here?

```suggestion
        nginx_cfg_filename = f"/var/snap/maas/current/http/regiond.nginx.conf"
```

Reply:
In addition, would it make sense to turn it into a Path and move it in a VARIABLE?